debug/bright-v7-types.json

```diff
@@ -0,0 +1,1324 @@
+{
+  "version": 7,
+  "name": "Bright",
+  "constants": {
+    "@name": { "type": "field", "value": "{name_en}" },
+    "@sans": { "type": "fontstack", "value": "Open Sans Regular, Arial Unicode MS Regular" },
+    "@sans_it": { "type": "fontstack", "value": "Open Sans Italic, Arial Unicode MS Regular" },
+    "@sans_md":{ "type": "fontstack", "value": "Open Sans Semibold, Arial Unicode MS Bold" },
+    "@sans_bd":{ "type": "fontstack", "value": "Open Sans Bold, Arial Unicode MS Bold" },
+    "@land": { "type": "color", "value": "#f8f4f0" },
+    "@water":{ "type": "color", "value":  "#a0c8f0" },
+    "@admin":{ "type": "color", "value":  "#9e9cab" },
+    "@admin-opacity": { "type": "opacity", "value": 0.5 },
+    "@park":     { "type": "color", "value": "#d8e8c8" },
+    "@cemetery": { "type": "color", "value": "#e0e4dd" },
+    "@hospital": { "type": "color", "value": "#fde" },
+    "@school":   { "type": "color", "value": "#f0e8f8" },
+    "@wood":     { "type": "color", "value": "#6a4" },
+    "@building": { "type": "color", "value": "#f2eae2" },
+    "@building_shadow": { "type": "color", "value": "#dfdbd7" },
+    "@building_color_transition": {
+        "type": "color",
+        "value": {
+          "base": 1,
+          "stops": [[15.5, "#f2eae2"], [16, "#dfdbd7"]]
+        }
+    },
+    "@building_opacity": {
+        "type": "color",
+        "value": {
+          "base": 1,
+          "stops": [[15, 0], [16, 1]]
+        }
+    },
+    "@aeroway": { "type": "color", "value": "#f0ede9" },
+    "@motorway": { "type": "color", "value": "#fc8" },
+    "@motorway_casing": { "type": "color", "value": "#e9ac77" },
+    "@motorway_tunnel": { "type": "color", "value": "#ffdaa6" },
+    "@main": { "type": "color", "value": "#fea" },
+    "@main_tunnel": { "type": "color", "value": "#fff4c6" },
+    "@street": { "type": "color", "value": "#fff" },
+    "@street_limited": { "type": "color", "value": "#f3f3f3" },
+    "@street_casing": { "type": "color", "value": "#cfcdca" },
+    "@path": { "type": "color", "value": "#cba" },
+    "@rail": { "type": "color", "value": "#bbb" },
+    "@text": { "type": "color", "value": "#334" },
+    "@text_halo": { "type": "color", "value": "rgba(255 },255 },255 },0.8)" },
+    "@marine_text": { "type": "color", "value": "#74aee9" },
+    "@marine_text_halo": { "type": "color", "value": "rgba(255 },255 },255 },0.7)" },
+    "@poi_text": { "type": "color", "value": "#666" },
+    "@poi_text_halo": { "type": "color", "value": "#ffffff" },
+    "@maki": { "type": "color", "value": "#666" },
+    "@point_translate": { "type": "dasharray", "value": [0, -30] },
```

Comment:
Hm, spec (https://github.com/mapbox/mapbox-gl-style-spec/pull/264/files#diff-8330e164d8515979491567746e4dc5c7R25) looks like `dash-array`, not `dasharray`, right?